### ISSUE TITLE
Confusion on Web Publishing

### ISSUE THREAD

**Issue by thruthesky** [FlutterFlow/flutterflow-issues#311]
## Expected behavior (required)

- There should be a link to open the published site after the `Web Publishing`.

## Current behavior (required)

- There is no link on the first publishing. The link appears when the developer refreshes.

## To Reproduce (required)

1. Create a project
2. Do `Web Publishing`

Please see the video.

### Bug Report Code (required, generate from Widget Tree)

N/A

## Context (required)

- Developers are lost what to do after the `Web Publish`. There should be a link to open the site after the first `Web Publish`.

## Screenshots / recordings

https://user-images.githubusercontent.com/5080329/210246143-541d754c-bf5b-4113-8d77-89ae770e5001.mp4

## Your environment

* Version of FlutterFlow used: v3.0 December 30, 2022 - Flutter 3.3.4
* Platform (e.g. Web, MacOS Desktop): Web
* Browser name and version: Google Chrome, Version 108.0.5359.124
* Operating system and version (desktop or mobile): iMac 2020, macOS Ventura 13.1

**Comment by thruthesky** [comment]
Hi, @organicnz I made a full video from Creating, Publishing, and Refreshing.

It's been a while that It didn't show the link after publish. And It really doesn't show for all the new projects that I created.
The link appears after refreshing the UI builder screen.

https://user-images.githubusercontent.com/5080329/210369021-61ef68cf-56e3-4ce6-b0f0-9d269578ef42.mp4

I made the project public.
https://app.flutterflow.io/project/web-publishing-w8epcz

Thank you.
